plugin.yml

```diff
@@ -128,4 +132,4 @@ permissions:
       myplot.claimplots.2:
         default: true
       myplot.claimplots.unlimited:
-        default: op
+        default: op
```

Comment:
line endings

Reply:
Which line endings does MyPlot use?

Reply:
I have my personal setup to view files in `CRLF` but git is set to only commit in `CR` endings

Reply:
actually this line fixes a mistake so we can probably leave it